registry-packages.ts

```diff
@@ -115,4 +115,14 @@ export const packages: PackageInfo[] = [
     },
     aliases: ['fujing'],
   },
+  {
+    name: '連加連乘',
+    repo: 'XingZiLong/continuous-operation',
+    description: 'Do continuous addition and mutiplication easily.',
+    author: {
+      name: 'XingZiLong',
+      url: 'httos://github.com/XingZiLong',
+    },
+    aliases: ['XingZiLong'],
```

Comment:
Something like this, 
```suggestion
    aliases: ['lian-jia-lian-cheng'],
```
or (or both)
```suggestion
    aliases: ['continuous-operation'],
```
You can change it to anything you want as long as it does not conflict with other packages. By setting this, users can install your package by 
```
wyg install lian-jia-lian-cheng
```
or 
```
wyg install 連加連乘
```

Reply:
Okay then, and am I finished?

Reply:
I don’t see my package registry in the file registry-packages.ts